code/client/munkiimport

```diff
@@ -760,6 +777,8 @@ def main():
 
     parser = PassThroughOptionParser(usage=usage, epilog=epilog)
 
+    parser.add_option('--init', action='store_true',
+                      help='Initialize a munki repository.')
```

Comment:
I am a bit conflicted about this option; people might do something silly like `munkiimport --init ~` and be confused about the results...